aws/network-primitives/scripts/create_domain_node_compose_file.sh

```diff
@@ -82,60 +76,39 @@ dsn_bootstrap_node_count=${4}
 enable_domains=${5}
 domain_id=${6}
 
-for (( i = 0; i < node_count; i++ )); do
-  if [ "${current_node}" != "${i}" ]; then
-    addr=$(sed -nr "s/NODE_${i}_MULTI_ADDR=//p" ~/subspace/node_keys.txt)
-    echo "      \"--reserved-nodes\", \"${addr}\"," >> ~/subspace/docker-compose.yml
-    echo "      \"--bootnodes\", \"${addr}\"," >> ~/subspace/docker-compose.yml
-  fi
-done
-
 for (( i = 0; i < bootstrap_node_count; i++ )); do
-  addr=$(sed -nr "s/NODE_${i}_MULTI_ADDR=//p" ~/subspace/bootstrap_node_keys.txt)
+  addr=$(sed -nr "s/NODE_${i}_MULTI_ADDR=//p" ~/subspace//bootstrap_node_keys.txt)
   echo "      \"--reserved-nodes\", \"${addr}\"," >> ~/subspace/docker-compose.yml
   echo "      \"--bootnodes\", \"${addr}\"," >> ~/subspace/docker-compose.yml
 done
 
+#// TODO: make configurable with gemini network as it's not needed for devnet
 for (( i = 0; i < dsn_bootstrap_node_count; i++ )); do
-  dsn_addr=$(sed -nr "s/NODE_${i}_MULTI_ADDR=//p" ~/subspace/dsn_bootstrap_node_keys.txt)
+  dsn_addr=$(sed -nr "s/NODE_${i}_SUBSPACE_MULTI_ADDR=//p" ~/subspace/dsn_bootstrap_node_keys.txt)
+  echo "      \"--dsn-reserved-peers\", \"${dsn)addr}\"," >> ~/subspace/docker-compose.yml
   echo "      \"--dsn-bootstrap-nodes\", \"${dsn_addr}\"," >> ~/subspace/docker-compose.yml
 done
 
-if [ ${reserved_only} == true ]; then
+if [ "${reserved_only}" == "true" ]; then
     echo "      \"--reserved-only\"," >> ~/subspace/docker-compose.yml
 fi
 
-if [ ${enable_domains} == true ]; then
+if [ "${enable_domains}" == "true" ]; then
     {
-    # system domain
-      echo '      "--",'
-      echo '      "--chain=${NETWORK_NAME}",'
-      echo '      "--validator",'
-      echo '      "--state-pruning", "archive",'
-      echo '      "--blocks-pruning", "archive",'
-      echo '      "--base-path", "/var/subspace/system_domain",'
-      echo '      "--keystore-path", "/var/subspace/keystore",'
-      echo '      "--rpc-cors", "all",'
-      echo '      "--rpc-port", "8933",'
-      echo '      "--ws-port", "8944",'
-      echo '      "--no-private-ipv4",'
-      echo '      "--unsafe-ws-external",'
-      echo '      "--relayer-id=${RELAYER_SYSTEM_ID}",'
-
     # core domain
       echo '      "--",'
       echo '      "--chain=${NETWORK_NAME}",'
+    #  echo '      "--alice",' //todo: make configurable with devnet
```

Comment:
we can remove this. We will never have --alice on any networks ever

Reply:
Great, will remove!